poll/poll.py

```diff
@@ -840,6 +840,41 @@ def generate_report_data(self, user_state_iterator, limit_responses=None):
             count += 1
             yield (user_state.username, report)
 
+    def index_dictionary(self):
+        """
+        Return dictionary prepared with module content and type for indexing.
+        """
+        # return key/value fields in a Python dict object
+        # values may be numeric / string or dict
+        # default implementation is an empty dict
+        xblock_body = super(PollBlock, self).index_dictionary()
+        answers = {
+            "answer_{}".format(answer_i): remove_markdown_and_html_tags(answer[1]['label'])
```

Comment:
Probably not that relevant, but given that this is a poll, these aren't answers, but options. 

Same for surveys below.